darts/models/forecasting/forecasting_model.py

```diff
@@ -950,6 +951,8 @@ def backtest(
         ----------
         series
             The (or a sequence of) target time series to use to successively train and evaluate the historical forecasts
+        historical_forecasts
```

Comment:
The method description above should be updated as well. Most of what it describes is only applied if historical_forecasts are not provided. Could you make the changes?